Runtimes/Core/cmake/modules/EmitSwiftInterface.cmake

```diff
@@ -26,5 +36,18 @@ function(install_swift_interface target)
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.private.swiftinterface"
       RENAME "${SwiftCore_MODULE_TRIPLE}.private.swiftinterface"
       DESTINATION "${SwiftCore_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
+
+    # Install catalyst interface files
+    if(SwiftCore_VARIANT_MODULE_TRIPLE)
+      install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${target}-${SwiftCore_VARIANT_MODULE_TRIPLE}/${target}.swiftmodule"
+        RENAME "${SwiftCore_VARIANT_MODULE_TRIPLE}.swiftmodule"
+        DESTINATION "${SwiftCore_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
+      install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${target}-${SwiftCore_VARIANT_MODULE_TRIPLE}/${target}.swiftinterface"
+        RENAME "${SwiftCore_VARIANT_MODULE_TRIPLE}.swiftinterface"
+        DESTINATION "${SwiftCore_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
```

Comment:
copy-pasta?

Reply:
Nevermind, the wrapping got me!

Reply:
No, the swift interfaces get installed into the swiftmodule directory.
The end result should be something like:

```
Swift.swiftmodule/<target>.swiftmodule
Swift.swiftmodule/<target>.swiftinterface
Swift.swiftmodule/<target>.private.swiftinterface
Swift.swiftmodule/<target-variant>.swiftmodule
Swift.swiftmodule/<target-variant>.swiftinterface
Swift.swiftmodule/<target-variant>.private.swiftinterface
```

The Destination is just the `Swift.swiftmodule/` directory. The `RENAME` sets the name to the triple.